docs/source/conf.py

```diff
@@ -39,19 +40,65 @@
 #
 # needs_sphinx = '1.0'
 
-# Add any Sphinx extension module names here, as strings. They can be
+# Add any Sphinx extension module names here as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    "sphinx.ext.intersphinx",
     "sphinx.ext.autodoc",
-    "sphinx_gallery.gen_gallery",
     "sphinx.ext.todo",
-    "sphinx.ext.napoleon",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.coverage",
+    "sphinx_copybutton",
+    "recommonmark",
+    "sphinx.ext.graphviz",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.inheritance_diagram",
+    "sphinx_gallery.gen_gallery",
+    "numpydoc",
     "pydata_sphinx_theme",
     "nbsphinx",
 ]
 
-autosummary_generate = True
```

Comment:
The config files for MAPDL and AEDT did not have this line and did have the Intersphinx mapping and numpydoc configuration info that I added.